package.json

```diff
@@ -14,7 +14,8 @@
     "@material/top-app-bar": "^2.3.0",
     "corejs-typeahead": "^1.2.1",
     "install": "^0.12.2",
-    "jquery": "^3.4.1"
+    "jquery": "^3.4.1",
+    "material-design-icons": "^3.0.1"
```

Comment:
ditto - we're using fontawesome.

material-design-icons doesn't work with older (but still in use) android.

Please put a note to this effect in the README by the frontend dev docs - sorry we didn't before.